### PR TITLE
header styles were updated

### PR DESCRIPTION
**Pull request by vladyslav78292**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -10,37 +10,37 @@ import burgerMenuIcon from 'assets/icons/burger-menu.svg'
...
+        <NavLink className={styles.header_shoppingbag_link} to="/shopping-bag">
+          <img className={styles.header_shoppingbag_icon} alt="Shopping Bag icon" src={logoBagIcon} />
         </NavLink>
-        <NavLink className="header_burger-menu_link" to="/burger-menu">
-          <img className="header_burger-menu_icon" alt="Shopping Bag icon" src={burgerMenuIcon} />
+        <NavLink className={styles.header_burgermenu_link} to="/burger-menu">
+          <img className={styles.header_burgermenu_icon} alt="Shopping Bag icon" src={burgerMenuIcon} />
         </NavLink>
       </div>
     </div>
```
Comment (Coyoote) [inline src/components/Header/Header.tsx:46]: add index.ts file for export in Header folder

File: src/components/Header/NavLinkBar/navLinkBar.tsx [hunk 2]
```diff
@@ -12,8 +13,8 @@ export const NavLinkBar: React.FC<Props> = ({ to, title }) => (
   <NavLink
     to={to}
     className={({ isActive }) =>
-      cn('navLink', {
-        activeLink: isActive,
+      cn(styles.navLink, {
+        [styles.activeLink]: isActive,
       })
     }
   >
```
Comment (Coyoote) [inline src/components/Header/NavLinkBar/navLinkBar.tsx:20]: rename file to NavLinkBar as your folder name

**Review by Coyoote — CHANGES_REQUESTED** [review]
Good job, left a small comment

File: src/App.tsx [hunk 3]
```diff
@@ -8,10 +8,10 @@ import { ProductCard } from 'components/ProductCard'
 function App() {
   return (
     <div className="App">
-      {/* <Header /> */}
-      {/* <PageNotFound /> */}
+      <Header />
```
Comment (vladyslav78292) [inline src/App.tsx:11]: in 4 row should be import { Header } from 'components/Header' , my fault

**Pull request merged**